src/_guides/language/concurrency/index.md

```diff
@@ -462,6 +458,42 @@ is slower when isolates are in different groups.
   Flutter doesn't support `Isolate.spawnUri()`.
 {{site.alert.end}}
 
+## Concurrency on the web
+
+All Dart apps can use `async-await`, `Future`, and `Stream`.
+Isolates, however, are not available on the [Dart web platform][].
+Dart web apps can use [web workers][] to
+run scripts in background threads
+similar to isolates.
+Web workers' functionality and capabilities
+differ somewhat from isolates, though.
+
+For instance, when web workers send data between threads,
+they copy the data back and forth.
+Data copying can be very slow, though,
+especially for large messages. 
+So, rather than copying, isolates _transfer_ the memory
+that holds the message in the exiting isolate
+to the receiving isolate
+(specifically by `Isolate.exit()`).
+
+{% comment %}
+Add ", and by extension `Isolate.run()`" to parenthesis after 2.19 and `run()` releases.
+{% endcomment %}
+
+How you create web workers and isolates also differs.
+On the web, you can only create web workers by declaring
+a separate program entrypoint and compiling that separately.
+Natively, you can start isolates in a similar manner,
+with separate sources, separate from existing isolates.
+More typically though, creation means branching from an existing isolate.
+This makes both isolates part of the same isolate group,
+and therefore much more lightweight 
+(as mentioned in [Performance and isolate groups](#performance-and-isolate-groups)).
```

Comment:
**question:** @lrhn : Does these sentences need to be removed, then?

Reply:
I'd say yes. It doesn't have any relevance to the web by itself, and it's not being compared or contrasted to any web feature.